@app/config/src/index.ts

```diff
@@ -3,11 +3,170 @@ const packageJson = require("../../../package.json");
 
 // TODO: customise this with your own settings!
 
-export const fromEmail =
-  '"PostGraphile Starter" <no-reply@examples.graphile.org>';
-export const awsRegion = "us-east-1";
-export const projectName = packageJson.name.replace(/[-_]/g, " ");
-export const companyName = projectName; // For copyright ownership
-export const emailLegalText =
+export const fromEmail: string = `"PostGraphile Starter" <no-reply@examples.graphile.org>`;
+export const awsRegion: string = "us-east-1";
+export const projectName: string = packageJson.name.replace(/[-_]/g, " ");
+export const companyName: string = projectName; // For copyright ownership
+export const emailLegalText: string =
   // Envvar here so we can override on the demo website
   process.env.LEGAL_TEXT || "<Insert legal email footer text here >";
+
+const required = (v: {
```

Comment:
A mini validation framework:
- `required`
- `defaultNumber`
- `minLength`
- `parseInteger`

Other options were using `envalid`, `io-ts`, etc. I think it's simple enough that isn't needed.

`name` is passed instead of being used to access the environment variables because some bundlers will replace `process.env.X` with the value, which would not work if we access `process.env` dynamically.